AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -798,33 +796,6 @@ - (void)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize
   }
 }
 
-- (void)_completeLayoutCalculation
-{
-  ASDN::MutexLocker l(_propertyLock);
-  [self calculatedLayoutDidChange];
-
-  // We generate placeholders at measureWithSizeRange: time so that a node is guaranteed to have a placeholder ready to go.
-  // This is also because measurement is usually asynchronous, but placeholders need to be set up synchronously.
-  // First measurement is guaranteed to be before the node is onscreen, so we can create the image async. but still have it appear sync.
-  if (_placeholderEnabled && [self _displaysAsynchronously] && self.contents == nil) {
-    
-    // Zero-sized nodes do not require a placeholder.
-    CGSize layoutSize = (_layout ? _layout.size : CGSizeZero);
-    if (CGSizeEqualToSize(layoutSize, CGSizeZero)) {
-      return;
-    }
-
-    if (!_placeholderImage) {
-      _placeholderImage = [self placeholderImage];
```

Comment:
This used to be called after any layout calculation, but at least by the method name, now only after layout transitions? (_pendingLayoutTransitionDidComplete)

This seems unexpected, let me know @maicki